### PR TITLE
DDP-5019 pex: default isStatus() to false

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

This PR changes the behavior of the `isStatus()` query in PEX so that it returns `false` instead of throwing exception when instance is not found.

I have looked through the configurations for existing studies (Angio, Brain, MBC, Osteo), and have walked through the studies locally to ensure that they work as expected.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.

## Testing

- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release

- [x] These changes require no special release procedures--just code!

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java [hunk 1]
```diff
@@ -260,14 +260,9 @@ private boolean applyFormPredicate(InterpreterContext ictx, FormPredicateContext
...
-
             return fetcher.findLatestActivityInstanceStatus(ictx, studyGuid, activityCode)
-                    .map(latestStatus -> expectedStatuses.contains(latestStatus))
-                    .orElseThrow(() -> {
-                        String msg = String.format("No activity instance of form %s found for user %s and study %s",
-                                activityCode, ictx.getUserGuid(), studyGuid);
-                        return new PexFetchException(new NoSuchElementException(msg));
-                    });
+                    .map(expectedStatuses::contains)
+                    .orElse(false);
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java:265]: This change will reduce a lot of noise from https://console.cloud.google.com/errors/CNq4jMrM5eyIEA?time=P30D&project=broad-ddp-prod.

File: study-builder/studies/angio/about-you.conf [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  include required("../snippets/activity-general-form.conf"),
```
Comment (yufengwng) [inline study-builder/studies/angio/about-you.conf:2]: Adding an `include` so these files pull in the defaults and don't break as easily.

Comment (ssettipalli) [reply study-builder/studies/angio/about-you.conf:2]: This only helps if study is redeployed (re-initialized) which might happen only in developers local at this stage. 
Doesn't hurt to have them though.

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Pull request merged**